playbook-website/app/views/pages/kit_show.html.erb

```diff
@@ -40,7 +40,7 @@
       <% if pb_doc_has_kit_type?(@kit, "react") %>
         <%= pb_rails("nav/item", props: { text: "React", link: kit_show_reacts_path(@kit), active: action_name == 'kit_show_react' }) %>
       <% end %>
-      <% if pb_doc_has_kit_type?(@kit, "swift") && Rails.env.development? %>
```

Comment:
checkout this comment. There are more env checks to remove 

https://github.com/powerhome/playbook/pull/2978#issuecomment-1866911417